package.json

```diff
@@ -23,13 +23,11 @@
   },
   "homepage": "https://github.com/mmarkelov/jest-playwright#readme",
   "scripts": {
-    "build": "rm -rf lib/ && babel src -d lib --ignore '**/*.test.js'",
+    "build": "tsc",
     "format": "prettier --write \"*.{js,md}\" \"src/**/*.js\"",
     "lint": "eslint .",
     "prepublishOnly": "yarn run build",
-    "test:bin": "jest src/bin/utils.test.js",
-    "test:utils": "jest src/utils.test.js",
-    "test": "npm run test:utils && npm run test:bin && jest tests"
+    "test": "jest src/bin/utils.test.ts tests/stub.test.ts && jest src/utils.test.ts"
```

Comment:
Shouldn't it be ```jest src/bin/utils.test.ts tests/stub.test.ts src/utils.test.ts```? To be more consistence

Reply:
By having all in the same jest statement, the run will fail. I've created a separate issue to fix this minor issue later: #46.

Reply:
@mxschmitt got it!